projects/core/src/user/facade/user.service.ts

```diff
@@ -58,8 +59,8 @@ export class UserService {
    *
    * @param userId
```

Comment:
I don't think we have a param anymore for this function

Reply:
Thanks! Good catch.